### DISCUSSION TITLE
Is it possible to undo the substitution of tabs in the output for spaces?

### DISCUSSION THREAD

**Discussion by dd** [Eugeny/tabby#8742] [Q&A] [answered]
I have several scripts that I use in my work to display data, I then copy and paste this data into a Google spreadsheet. However, this does not work in Tabby, because Tabby replaces tabs with spaces and when pasting copied data into a table, the data is not split into columns.

![image](https://github.com/Eugeny/tabby/assets/2998061/57fc18e7-2d6a-4e9e-9254-d9af400f496d)

**Comment by Eugeny** [comment, marked as answer]
It's not possible unfortunately as xterm, the underlying renderer, uses a character grid internally.
